plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/sidebar-navigation-screen-footer.tsx

```diff
@@ -2,21 +2,84 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { createInterpolateElement } from '@wordpress/element';
+import {
+	createInterpolateElement,
+	useCallback,
+	useContext,
+	useEffect,
+} from '@wordpress/element';
 import { Link } from '@woocommerce/components';
+import { Spinner } from '@wordpress/components';
+// @ts-expect-error Missing type in core-data.
+import { __experimentalBlockPatternsList as BlockPatternList } from '@wordpress/block-editor';
+
 /**
  * Internal dependencies
  */
 import { SidebarNavigationScreen } from './sidebar-navigation-screen';
 import { ADMIN_URL } from '~/utils/admin-settings';
+import { useEditorBlocks } from '../hooks/use-editor-blocks';
+import { usePatternsByCategory } from '../hooks/use-pattern';
+import { HighlightedBlockContext } from '../context/highlighted-block-context';
+
+const footerPatternNames = [
+	'woocommerce-blocks/footer-large',
+	'woocommerce-blocks/footer-large-dark',
+	'woocommerce-blocks/footer-simple',
+	'woocommerce-blocks/footer-simple-dark',
+	'woocommerce-blocks/footer-simple-menu-and-cart',
+	'woocommerce-blocks/footer-with-2-menus',
+	'woocommerce-blocks/footer-with-2-menus-dark',
+	'woocommerce-blocks/footer-with-3-menus',
```

Comment:
Could you please update this list to only include these? We only need to show them. 🙂 

- woocommerce-blocks/footer-large
- woocommerce-blocks/footer-simple-menu-and-cart	
- woocommerce-blocks/footer-with-3-menus

https://github.com/woocommerce/woocommerce/issues/39719#issuecomment-1718882324

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/40136/commits/53a6cd884eb3daf2c633aaff4cdff5bc959cf782